src/components/modules/banners/ausMomentContributionsBanner/AusMomentContributionsBanner.tsx

```diff
@@ -459,7 +459,7 @@ const chevronDown = (
     </svg>
 );
 
-const messageSupporter = (goalReached: boolean) => {
+const messageSupporter = (goalReached: boolean): React.ReactElement => {
```

Comment:
these should actually just be functional components, but I don't think there's much point doing that refactor

Reply:
we can also delete old banner code like this. It's easy to find with github if we need it again